p2p/stream/protocols/sync/const.go

```diff
@@ -19,11 +19,11 @@ const (
 
 	// GetNodeDataCap is the cap of request of single GetNodeData request
 	// This number has an effect on maxMsgBytes as 20MB defined in github.com/harmony-one/harmony/p2p/stream/types.
-	GetNodeDataCap = 10
+	GetNodeDataCap = 256
```

Comment:
were there any analysis done on the change of this cap ? is the comment `// This number has an effect on maxMsgBytes as 20MB defined in github.com/harmony-one/harmony/p2p/stream/types.` still applicable ?

Reply:
I chose a maximum of 256 for the number of states data per request because the states themselves are relatively small in size. Assuming an average state size of 1 kilobyte, when we request 256 states, the total data size amounts to 256 kilobytes. By keeping the number of states per request at 256, we ensure that the data size remains well below 20 megabytes (20MB), which is a reasonable and manageable size for data transfer and processing. This number for other chain like Ethereum is larger. Later we can increase this number again.

Reply:
thanks